specification/billingbenefits/resource-manager/Microsoft.BillingBenefits/stable/2022-11-01/billingbenefits.json

```diff
@@ -73,8 +73,11 @@
     },
     "/providers/Microsoft.BillingBenefits/savingsPlanOrderAliases/{savingsPlanOrderAliasName}": {
       "put": {
-        "description": "Create a savings plan.",
+        "description": "Create a savings plan. Savings plan can only be purchased with certain permissions. Learn more about permissions at https://go.microsoft.com/fwlink/?linkid=2215851",
```

Comment:
We should just say: `Create a savings plan. Learn more about permissions at https://go.microsoft.com/fwlink/?linkid=2215851`